spec.emu

```diff
@@ -121,7 +124,7 @@
               1. Return _promiseCapability_.[[Promise]].
             1. Let _assertionsObj_ be Completion(Get(_options_, *"with"*)).
             1. IfAbruptRejectPromise(_assertionsObj_, _promiseCapability_).
-            1. [normative-optional-legacy=""] If _assertionsObj_ is *undefined*, then
+            1. [deprecated=""] If _assertionsObj_ is *undefined*, then
```

Comment:
This isn't a "deprecated subclause". We should probably just have a NOTE step here that gets replaced in a deprecated subclause, much like what we do in Annex B. Search for "replaces-step" for examples to copy.

Reply:
I was following the path introduced by https://github.com/tc39/ecma262/pull/2952, which inlines Annex B changes to the algorithms rather than replacing NOTE steps out-of-band.

I could update the description of "deprecated" to be "deprecated subclause, step, or production", or I could update this step to be
```
1. If the host supports the deprecated assert keyword for import attributes and if _assertionsObj_ is *undefined*, then
```
with "assert keyword for import attributes" linking to an actual deprecated emu-clause that explains what it is.

What do you think about these two options? I would also be happy to join an editors call if needed.

Reply:
Wdyt about this?
<img width="927" alt="image" src="https://user-images.githubusercontent.com/7000710/229229434-d8572046-e03d-4a5b-9933-40686ece2fcc.png">